packages/wallet/ui/src/service/Issuers.js

```diff
@@ -0,0 +1,44 @@
+import { makeNotifierKit } from '@agoric/notifier';
+
+export const getIssuerService = signSpendAction => {
+  const suggestions = new Map();
+  const { notifier, updater } = makeNotifierKit();
```

Comment:
not worth changing right now but for the future consider `makePublishKit`

Reply:
Added TODO